compose.yaml

```
@@ -220,9 +234,27 @@ services:
+        target: /data
+        type: volume
+
 # https://docs.docker.com/reference/compose-file/volumes/
 volumes:
   caddy_config:
   caddy_data:
   minio_data:
-  postgres_data:
\ No newline at end of file
+  postgres_data:
+  redis_data:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)